src/main/java/com/synopsys/integration/detect/util/bdio/protobuf/DetectProtobufBdioHeaderUtil.java

```diff
@@ -9,25 +9,26 @@
 import com.blackducksoftware.bdio.proto.ProtobufBdioWriter;
 import com.blackducksoftware.bdio.proto.api.BdioHeader;
 import com.synopsys.integration.detect.util.DetectZipUtil;
+import com.synopsys.integration.util.NameVersion;
 
-public class DetectProtobufBdioUtil {
-    private static final String TEMP_BDIO_ARCHIVE_PATH = "target/bdio-protobuf.zip";
-
+public class DetectProtobufBdioHeaderUtil {
     private final String scanId;
     private final String scanType;
+    private final NameVersion projectNameVersion;
 
-    public DetectProtobufBdioUtil(String scanId, String scanType) {
+    public DetectProtobufBdioHeaderUtil(String scanId, String scanType, NameVersion projectNameVersion) {
         this.scanId = scanId;
         this.scanType = scanType;
+        this.projectNameVersion = projectNameVersion;
     }
 
-    public File createProtobufBdioHeader() throws IOException {
+    public File createProtobufBdioHeader(File targetDirectory) throws IOException {
         BdioHeader bdioHeader = new BdioHeader(
             scanId,
             scanType,
             "codeLocation name",
```

Comment:
Correct value?

Reply:
This value is not needed by scan service as of now to create a container scan & return a scan ID. Also, since we are not scanning a project but an image artifact, this value can be an empty string.